project/Dependencies.scala

```diff
@@ -19,4 +20,6 @@ object Dependencies {
   val `zio-streams`  = "dev.zio" %% "zio-streams"  % ZioVersion
   val `zio-test`     = "dev.zio" %% "zio-test"     % ZioVersion % "test"
   val `zio-test-sbt` = "dev.zio" %% "zio-test-sbt" % ZioVersion % "test"
+
+  val reflect = Def.map(scalaVersion)("org.scala-lang" % "scala-reflect" % _)
```

Comment:
Why do we need this?

Reply:
we need it for scala 2.x macro implementation